src/_layouts/docs.html

```diff
@@ -1,74 +1,83 @@
+        {% endfor %}
+      </ul>
+    </nav>
+    <form class="Docs-search js-only" action="/search" method="GET">
+      <input class="Docs-search-input" type="search" name="q" id="q" placeholder="Search Docs…" />
     </form>
-    <ul class="subnav-links">
-      {% for link in site.data.docs.subnav %}
-      <li><a class="sidebar-link {% if page_href == link.href %}sidebar-link-active{% endif %}" href="/{{link.href | escape}}">{{link.title | escape}}</a>
-      {% endfor %}
-    </ul>
-  </nav>
+  </header>
 
-  <nav class="sidebar" aria-label="Primary">
-    <button class="sidebar-opener u-showOnlyOnDocsMobile"
-         onclick="document.querySelector('.sidebar').classList.toggle('is-open')">
-      Contents
+  <div class="Docs-body">
+    <aside class="Docs-sidebar  Docs-sidebar--defaultLayout">
+      <nav class="Docs-subnav">
+        <dl class="js-accordion Docs-subnav-list" data-accordion-prefix-classes="Docs-subnav">
+        {% for section in site.data.docs.sidebar[page.sidebar] %}
+          {% assign pages = section.items %}
 
-      {% svg assets/svg/carrot-down.svg %}
-    </button>
+          {% if section.default and page_href == '' %}
+            {% assign active_section = true %}
+          {% else %}
+            {% assign active_section = pages | find: "href", page_href %}
+          {% endif %}
 
-    <div class="sidebar-contents js-accordion" data-accordion-prefix-classes="nav">
-    {% for section in site.data.docs.sidebar %}
-      {% if section.include == 'examples' %}
-        {% assign pages = site.data.examples %}
-      {% else %}
-        {% assign pages = section.items %}
-      {% endif %}
+          {% if active_section %}
+            {% assign current_section = section %}
+            {% assign current_section_pages = pages %}
+          {% endif %}
 
-      {% if section.default and page_href == '' %}
-        {% assign active_section = true %}
-      {% else %}
-        {% assign active_section = pages | find: "href", page_href %}
-      {% endif %}
-
-      {% if active_section %}
-        {% assign current_section = section %}
-        {% assign current_section_pages = pages %}
-      {% endif %}
-
-      <h3 class="sidebar-sectionTitle js-accordion__header" {% if active_section %}data-accordion-opened="true"{% endif %}>
-        {{ section.title | escape }}
-      </h3>
-
-      <ul class="sidebar-sectionList js-accordion__panel">
-        {% for page in pages %}
-          <li><a class="sidebar-link {% if page_href == page.href %}sidebar-link-active{% endif %}" href="/{{page.href | escape}}">{{page.title | escape}}</a>
-        {% endfor %}
-      </ul>
-
-    {% endfor %}
-    </div>  
-
-  </nav>
-  <article class="docsContent">
-    <div class="docsHeader">
+          {% assign pages = section.items %}
+            <dt class="js-accordion__header Docs-subnav-title" data-accordion-opened="true">
+                {{ section.title | escape }}
+                {% svg assets/svg/carrot-down.svg %}
```

Comment:
FYI clicking ON the caret icon isn't triggering an accordion expand/collapse

Reply:
Nice catch! Fixed.